index.css

```
@@ -152,7 +152,7 @@ body {
   display: none;
 }
 
-#touchControls:not([data-ui-mode='COMMAND']):not([data-ui-mode='FIGHT']):not([data-ui-mode='BALL']):not([data-ui-mode='TARGET_SELECT']) #apad .apadBattle {
+#touchControls:not([data-ui-mode='COMMAND']):not([data-ui-mode='FIGHT']):not([data-ui-mode='BALL']):not([data-ui-mode='TARGET_SELECT']):not([data-ui-mode='MODIFIER_SELECT']) #apad .apadBattle {
```

since CSS4 not can be used with multiple params to chain them 

```suggestion
#touchControls:not([data-ui-mode='COMMAND'], [data-ui-mode='FIGHT'], [data-ui-mode='BALL'], [data-ui-mode='TARGET_SELECT'], data-ui-mode='MODIFIER_SELECT']) #apad .apadBattle {
```
**untested!**

source: https://developer.mozilla.org/en-US/docs/Web/CSS/:not?ref=serpapi

---

Tested on FF and it works; however: Opera (Android) doesn't seem to support that function, according to your reference link.
So it might not be worth it to improve the code style...

---

that is true. Not worth it then. MB